src/main/java/com/kireyko/movieland/util/JsonJacksonConverter.java

```diff
@@ -0,0 +1,43 @@
+package com.kireyko.movieland.util;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.kireyko.movieland.entity.Movie;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+
+@Service
```

Comment:
why this's a Service? Why this's a bean at all? I think abstract class with static methods would be enough